client/state/data-layer/wpcom/sites/memberships/index.js

```diff
@@ -57,6 +55,23 @@ export const handleMembershipsList = dispatchRequestEx( {
 	onError: noop,
 } );
 
+export const handleSubscribedMembershipsList = dispatchRequestEx( {
+	fetch: action =>
+		http(
+			{
+				method: 'GET',
+				path: '/me/memberships/subscriptions',
+			},
+			action
+		),
+	onSuccess: ( {}, { subscriptions, total } ) => ( {
+		type: MEMBERSHIPS_SUBSCRIPTIONS_RECEIVE,
+		subscriptions,
+		total,
+	} ),
```

Comment:
`_RECEIVE` is also a pattern we want to avoid but I think we should talk again after this lands and discuss `httpData` - I promise you will like it